action.yml

```diff
@@ -42,33 +42,83 @@ outputs:
 runs:
   using: "composite"
   steps:
-    - name: Fetch project metadata
-      id: fetch_project_metadata
+    - name: Fetch content metadata
+      id: fetch_content_metadata
       env:
         GITHUB_TOKEN: ${{ inputs.github_token }}
-        PROJECT_NUMBER: ${{ inputs.project_number }}
-        ORGANIZATION: ${{ inputs.organization }}
         FIELD: ${{ inputs.field }}
         CONTENT_ID: ${{ inputs.content_id }}
-        VALUE: ${{ inputs.value }}
-        FILE_NAME: project-${{ inputs.organization }}-${{ inputs.project_number }}.json
+        FILE_NAME: project-${{ inputs.organization }}-${{ inputs.project_number }}_content.json
```

Comment:
Since the query is now specific to the issue, we'll need to update the filename to include the issue number (and repository?). This functionality is intended to "cache" the metadata where multiple fields are updated for the same issue in a single call.